api/src/lib/helpers.ts

```diff
@@ -914,3 +914,11 @@ export const createPublicationFooterTemplate = (publication: I.Publication): str
 };
 
 export const isEmptyContent = (content: string): boolean => (content ? /^(<p>\s*<\/p>)+$/.test(content) : true);
+
+export const checkEnvVariable = (variable: string | undefined): string => {
+    if (variable === undefined) {
+        throw new Error(`Environment Variable ${variable} is undefined.`);
```

Comment:
if the env variable is undefined the error message will say: 
_Environment Variable undefined is undefined._

Reply:
I get this when the variable is undefined Environment Variable 'Queue URL' is undefined.?